src/mutation.cpp

```diff
@@ -481,6 +482,41 @@ void player::activate_mutation( const trait_id &mut )
         }
         tdata.powered = false;
         return;
+    } else if( mut == trait_SLIMESPAWNER2 ) {
```

Comment:
This whole block is copypasted section above. This is totally wrong.
Two options:
* Extract a function and call it here and above
* `} else if( mut == trait_SLIMESPAWNER || mut == trait_SLIMESPAWNER2 ) {` and then later pick summon monster or summon npc based on `mut`